bps/resources/HSC/DRP-RC2.yaml

```diff
@@ -53,8 +53,12 @@ pipetask:
     requestMemory: 5000
   consolidateForcedSourceTable:
     requestMemory: 200000
+  makeVisitTable:
+    requestMemory: 5400
+  makeCcdVisitTable:
+    requestMemory: 5400
 
-  # faro tasks
+# faro tasks
```

Comment:
You made an unnecessary whitespace change here.  Please amend this commit so that it does not change this line.